### PR TITLE
Detecting android phones and adding swipe events

### PR DESCRIPTION
**Pull request by BC-Mikey**
FORD-60: added detection for android tablets & smartphones, when smartphones are detected, use touch events for playlist

### CODE REVIEW

File: lib/bcc_web.js [hunk 1]
```diff
@@ -47,6 +47,12 @@ v1.0.0: 10/29/2014
             Android: function() {
                 return navigator.userAgent.match(/Android/i);
             },
+            AndroidPhone: function() {
+                return navigator.userAgent.match(/(?=.*Android)(?=.*Mobile)/i);
+            },
+            AndroidTablet: function() {
+                return navigator.userAgent.match(/(?=.*Android)(?!Mobile)/i);
+            },
```
Comment (bceacuna) [inline lib/bcc_web.js]: Did you test this regex? This regex returns a match when the text contains "Mobile" and it shouldn't

Mozilla/5.0 (Linux; Android 4.0.4; Galaxy Nexus Build/IMM76B) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.133 Safari/535.19 Mobile

Comment (BC-Mikey) [inline lib/bcc_web.js]: I see what you're saying now. I'll change it. Thanks.

**Comment by BC-Mikey** [comment]
Fixed issue, let me know when it is ok to merge.

File: lib/bcc_web.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ v1.0.0: 10/29/2014
             Android: function() {
                 return navigator.userAgent.match(/Android/i);
             },
+            AndroidPhone: function() {
+                return navigator.userAgent.match(/(?=.*Android)(?=.*Mobile)/i);
+            },
+            AndroidTablet: function() {
+                return navigator.userAgent.match(/(?=.*Android)(?!Mobile)/i) && !navigator.userAgent.match(/Mobile/i);
+            },
```
Comment (bceacuna) [inline lib/bcc_web.js]: I believe you can simplify this to
return navigator.userAgent.match(/(?=.*Android)/i) && !navigator.userAgent.match(/Mobile/i);

**Comment by bceacuna** [comment]
looks good

File: lib/bcc_web.js (hunk 2 above)
Comment (BC-Mikey) [inline lib/bcc_web.js]: Woops thought I did but only did in my test environment. Thanks!

**Comment by BC-Mikey** [comment]
Merging.

**Pull request merged**